### PR TITLE
Fix school urn to be in the correct format in sandbox seed data

### PR DESCRIPTION
**Pull request by plisovin**
And attach cohort to early career teacher profile

### CODE REVIEW

**Review by kicferk1 — APPROVED** [review]

**Comment by kicferk1** [comment]
Looks like linter is unhappy.

> ### Git hooks
> Run `git config core.hooksPath .githooks` to use the included git hooks. They run linting on commit and check for AWS secrets.

From our readme, it is supposed to run linter in a pre-commit hook, so you don't get this problem (or at least so you don't get it nearly as often).

**Comment by plisovin** [comment]
> Looks like linter is unhappy.
> 
> > ### Git hooks
> > Run `git config core.hooksPath .githooks` to use the included git hooks. They run linting on commit and check for AWS secrets.
> 
> From our readme, it is supposed to run linter in a pre-commit hook, so you don't get this problem (or at least so you don't get it nearly as often).

thanks, didn't have it

File: db/seeds/sandbox_data.rb [hunk 1]
```diff
@@ -18,15 +18,16 @@ def generate_provider_token(lead_provider, cohort, logger)
...
-        address_line1: Faker::Address.street_address,
-        postcode: Faker::Address.postcode,
-      )
+      school = School.find_or_create_by!(
+        urn: sprintf("%06d", (10_000 + i)),
+      ) do |s|
+        s.name = Faker::Company.name
+        s.address_line1 = Faker::Address.street_address
+        s.postcode = Faker::Address.postcode
+      end
```
Comment (ethanmills) [inline db/seeds/sandbox_data.rb:30]: These are URNs for real schools

File: db/seeds/sandbox_data.rb [hunk 2]
```diff
@@ -18,15 +18,16 @@ def generate_provider_token(lead_provider, cohort, logger)
...
       user = User.create!(full_name: Faker::Name.name, email: Faker::Internet.email)
 
-      school = School.create!(
-        urn: Faker::Internet.uuid,
-        name: Faker::Company.name,
-        address_line1: Faker::Address.street_address,
-        postcode: Faker::Address.postcode,
-      )
+      school = School.find_or_create_by!(
+        urn: sprintf("%06d", (10_000 + i)),
```
Comment (ethanmills) [inline db/seeds/sandbox_data.rb:25]: Are we not going to end up with the same URNs for schools for different providers?

Comment (plisovin) [reply db/seeds/sandbox_data.rb]: you are right, fixed in 059015e

**Comment by timabell** [comment]
Fixes https://dfedigital.atlassian.net/browse/CPDTP-182

**Pull request merged**